osier/models/dispatch.py

```diff
@@ -548,6 +548,8 @@ def _write_model_equations(self):
 
     def _format_results(self):
         df = pd.DataFrame(index=self.model.Time)
+        expr = np.array(sum(self.model.VariableCost[g, t] * self.model.x[g, t].value
+                   for g in self.model.Generators) for t in self.model.Time)
```

Comment:
Do you se this variable anywhere other than line 566? If you don't then why not just define the `df['Cost']` as this expression directly?

Reply:
I suppose that would be fine, though this feels better to me. If you have a strong preference, I’ll change it.

Reply:
My preference would be to just define the `df['Cost']` with this expression directly.